plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -1399,34 +1410,94 @@ def exception_handler(module, ex, exit_args, one_name):
                 if "errors" in argspec.args:
                     handlers_user_args["errors"] = _errors
 
+        debug = 0x10
+        if debug & 0x1:
```

Comment:
What about calling these flags `IPA_DBG_CMD_LIST` and `IPA_DBG_CMD_COUNT` and make them class members?

Reply:
I added DEBUG_COMMAND_ constants for this.